contracts/LightClient.sol

```diff
@@ -35,57 +35,85 @@ contract LightClient is ILightClient, Ownable {
         uint256 _height,
         bytes32
     ) public virtual override onlyOwner {
-        _createEpochValidators(1, _height, _epochSigners, _epochVotingPowers);
+        uint256 epochIdx = _deriveEpochIdx(_height, epochPeriod);
```

Comment:
I think it is better to impose `_height = 0` and `epochIdx = 1` upon initialization?